builder/build-image.sh

```diff
@@ -43,7 +43,7 @@ UPLOAD="$4"
 # create a virtual env for fedcloudclient
 python3 -m venv "$PWD/.venv"
 export PATH="$PWD/.venv/bin:$PATH"
-pip install -qqq fedcloudclient simplejson yq python-hcl2 IM-client
+pip install -qqq fedcloudclient simplejson yq python-hcl2 IM-client>=1.8.2
```

Comment:
As suggested by @brucellino, we should move this to a requirements.txt, pin them and have it properly managed

Reply:
Sure, let's capture this: https://github.com/EGI-Federation/fedcloud-vmi-templates/issues/147

And perhaps we can have a dedicated PR to address it